Source/WebCore/platform/graphics/ComplexTextController.cpp

```diff
@@ -793,6 +809,7 @@ ComplexTextController::ComplexTextRun::ComplexTextRun(const Font& font, const UC
     , m_indexEnd(indexEnd)
     , m_stringLocation(stringLocation)
     , m_isLTR(ltr)
+    , m_textAutospaceSize(0.125 * font.fontMetrics().ideogramWidth().value_or(0))
```

Comment:
What is this magic number `0.125` ? We should define a name for this magic constant.